x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/layout.tsx

```diff
@@ -67,11 +72,15 @@ export const CreatePackagePolicyPageLayout: React.FunctionComponent<{
             <EuiFlexItem>
               <EuiText>
                 <h1 data-test-subj={`${dataTestSubj}_pageTitle`}>
-                  {from === 'edit' || from === 'package-edit' ? (
+                  {from === 'edit' ||
+                  from === 'package-edit' ||
+                  from === 'upgrade-from-fleet-policy-list' ||
+                  from === 'upgrade-from-integrations-policy-list' ? (
                     <FormattedMessage
                       id="xpack.fleet.editPackagePolicy.pageTitleWithPackageName"
-                      defaultMessage="Edit {packageName} integration"
+                      defaultMessage="{action} {packageName} integration"
```

Comment:
I think it will be better to have two i18n key and defaultMessage here `xpack.fleet.editPackagePolicy.editPageTitleWithPackageName` and ``xpack.fleet.editPackagePolicy.upgradePageTitleWithPackageName` it probably will allow better translations